kazoo/tests/test_cache.py

```diff
@@ -111,25 +111,25 @@ def test_start(self):
         self.wait_cache(since=TreeEvent.INITIALIZED)
 
         stat = self.client.exists(self.path)
-        eq_(stat.version, 0)
+        assert stat.version == 0
 
-        eq_(self.cache._state, TreeCache.STATE_STARTED)
-        eq_(self.cache._root._state, TreeNode.STATE_LIVE)
+        assert self.cache._state == TreeCache.STATE_STARTED
+        assert self.cache._root._state == TreeNode.STATE_LIVE
 
-    @raises(KazooException)
     def test_start_started(self):
         self.make_cache()
-        self.cache.start()
+        with pytest.raises(KazooException):
+            self.cache.start()
 
-    @raises(KazooException)
     def test_start_closed(self):
         self.make_cache()
-        self.cache.start()
+        # self.cache.start()
```

Comment:
Broken test, always triggered here and thus no different than above

Reply:
Good catch